include/hip/hip_ext.h

```diff
@@ -142,7 +144,7 @@ inline
 void hipExtLaunchKernelGGL(F kernel, const dim3& numBlocks,
                            const dim3& dimBlocks, std::uint32_t sharedMemBytes,
                            hipStream_t stream, hipEvent_t startEvent,
-                           hipEvent_t stopEvent, std::uint32_t flags,
+                           hipEvent_t stopEvent, std::uint32_t flags = hipExtInOrderLaunch,
```

Comment:
Should we also have a doxygen block here defining this, so that users know how to use this function? Especially how to call the flags variable.